src/meltano/cli/install.py

```diff
@@ -52,24 +51,18 @@ def install(project, plugin_type, plugin_name, clean, parallelism):
 
     plugins_service = ProjectPluginsService(project)
 
-    if plugin_type:
-        try:
+    try:
+        if plugin_type:
             plugin_type = PluginType.from_cli_argument(plugin_type)
-        except ValueError:
```

Comment:
This was also a dumb bug on my part - i should have covered the `get_plugins_of_type` call with the exception out of the gate 😮‍💨